ank/src/main.rs

```diff
@@ -66,8 +66,9 @@ async fn main() {
                     args.server_url,
                 )
                 .await;
-                if let Some(out_text) = cmd.get_workloads(agent_name, state, workload_name).await {
-                    println!("{}", out_text);
+                match cmd.get_workloads(agent_name, state, workload_name).await {
+                    Ok(out_text) => println!("{}", out_text),
+                    Err(error) => log::error!("Failed to get workloads: '{}'", error),
```

Comment:
This is rather general question to the error handling in ```ank```. Do we want to trace this error as log, or as a plain text?

Reply:
Yes, this is not correct. I made some corrections in main.rs now, but as discussed, we need a complete new concept for handling logs and error messages/exit codes in the ank CLI. 
I created a ticket for this: #49. Maybe you can have a look inside.

Reply:
The new ticket look good. I agree to continue with this topic there.